### ISSUE TITLE
Add support for Nodejs22.x

### ISSUE THREAD

**Issue by JamesKyburz** [aws-cloudformation/cfn-lint#3837]
### CloudFormation Lint Version

1.20.0

### What operating system are you using?

Mac

### Describe the bug

E3030 'nodejs22.x' is not one of ['dotnet6', 'dotnet8', 'dotnetcore1.0', 'dotnetcore2.0', 'dotnetcore2.1', 'dotnetcore3.1', 'go1.x', 'java11', 'java17', 'java21', 'java8', 'java8.al2', 'nodejs', 'nodejs10.x', 'nodejs12.x', 'nodejs14.x', 'nodejs16.x', 'nodejs18.x', 'nodejs20.x', 'nodejs4.3', 'nodejs4.3-edge', 'nodejs6.10', 'nodejs8.10', 'provided', 'provided.al2', 'provided.al2023', 'python2.7', 'python3.10', 'python3.11', 'python3.12', 'python3.13', 'python3.6', 'python3.7', 'python3.8', 'python3.9', 'ruby2.5', 'ruby2.7', 'ruby3.2', 'ruby3.3']
sam-latest-runtime-check.yaml:4:3

Reported to sam cli too [here too](https://github.com/aws/aws-sam-cli/issues/7726) to co-ordinate.

### Expected behavior

Validation to pass

### Reproduction template

```yaml
AWSTemplateFormatVersion: "2010-09-09"
Transform: AWS::Serverless-2016-10-31
Resources:
  Function:
    Type: AWS::Serverless::Function
    Properties:
      Runtime: nodejs22.x
      Handler: index.main
      InlineCode: |
        export async function main() {}
```

**Comment by kddejong** [comment]
Fix released in https://github.com/aws-cloudformation/cfn-lint/releases/tag/v1.20.1

**Comment by animaxcg** [comment]
This is an issue again in 1.32.2

**Comment by kddejong** [comment]
This cannot be replicated in v1.32.2.  Are you sure you are using that version?
